grafana/standard_dashboards/Network_Details.json

```diff
@@ -0,0 +1,1024 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"a
+      }
+    ]
+  },
+  "description": "",
+  "editable": false,
+  "gnetId": 1856,
+  "graphTooltip": 2,
+  "id": 15,
+  "iteration": 1596489115868,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "PROMETHEUS",
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 7,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "hiddenSeries": false,
+      "id": 29,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "options": {
+        "dataLinks": []
+      },
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "node_load1{job=~\"[[osnodes]]\"}",
```

Comment:
Probably just not me knowing any better, but node_load? Like, system load?

Reply:
Yes, that's just the name of the metric in node exporter for the 1/5/15 minute system load metrics

Reply:
The dashboard I took the network metrics from included system load in it. It is duplicated from the OS Details dashboard, but I figured network load could coincide with system load periods and it could make it easier to correlate issues if they're on the same page.